packages/shared-influx-pieces/src/register.ts

```diff
@@ -0,0 +1,7 @@
+import { ListenerStore, container } from '@skyra/http-framework';
+import { registerSchedule } from 'lib/utils/schedule.js';
+import { resolve } from 'path';
+
+container.stores.register(new ListenerStore().registerPath(resolve('listeners')));
+
+registerSchedule().catch();
```

Comment:
I meant no catch at all, plus, not only it's a fully sync function with no way of throwing, that `.catch` is useless. And we shouldn't have hanging promises either way.
```suggestion
registerSchedule();
```